File: 9041 - Software Instruction/lab06/echon.py
#!/usr/bin/python

import sys


if (len(sys.argv)!=3):
	print "Usage: ./echon.py <number of lines> <string>"
	exit()

number=sys.argv[1]
if not number.isdigit() and number > 0:
	print "./echon.py: argument 1 must be a non-negative integer"
	exit()
string=sys.argv[2]
for i in range(0,int(number)):
	print string
	

File: 9041 - Software Instruction/lab05/echon.pl
#!/usr/bin/perl -l

my $scalar = @ARGV;
my $number = @ARGV[0];
my $string = @ARGV[1];

if ($scalar != 2) {
	print "Usage: ./echon.pl <number of lines> <string>";
	exit 1;
}

if ($number !~ /^[0-9]*$/) {
	print "./echon.pl: argument 1 must be a non-negative integer";
	exit 1;
}

foreach my $i(1..$number){
	print "$string";
}
